docs/03-Filtering Patches.md

```diff
@@ -1,18 +1,51 @@
 ## Filtering Patches
```

Comment:
Before line 3  there should be some introduction why is this relevant. For example:

> Palindrom deeply observes changes made to the `palindrom.obj` object and converts them to patches.  If you have a need to ignore some kind of changes, there are two ways to do it. Either make the changes in non-enumerable property or use a filtering function to ignore changes made in the enumerable properties.